modules/installation-initializing-manual.adoc

```diff
@@ -144,16 +144,26 @@ it in the `<installation_directory>`.
 ====
 You must name this configuration file `install-config.yaml`.
 ====
+
 ifdef::restricted,restricted-upi[]
+
 ** Unless you use a registry that {op-system} trusts by default, such as
 `docker.io`, you must provide the contents of the certificate for your mirror
 repository in the `additionalTrustBundle` section. In most cases, you must
 provide the certificate for your mirror.
 ** You must include the `imageContentSources` section from the output of the command to
 mirror the repository.
+
+[IMPORTANT]
+====
+** The 'ImageContentSourcePolicy' is generated as an output of `oc mirror` after the mirroring process is finished.
+** The `oc mirror` command generates a ImageContentSourcePolicy' file which contains the YAML needed to define 'ImageContentSourcePolicy'. 
```

Comment:
The second _ImageContentSourcePolicy_ needs a noun. I see the term [resource](https://github.com/openshift/openshift-docs/pull/63332/files#diff-57166dbcd99d39b77ac269edbafbd33682c7a4ef400cf2fc3f12fe7b2c3cb57dR42) below.
```suggestion
** The `oc mirror` command generates an 'ImageContentSourcePolicy' file, which contains the YAML needed to define the 'ImageContentSourcePolicy' resource. 
```